good-enough-practices-for-scientific-computing.tex

```diff
@@ -1237,68 +1251,59 @@ \subsection*{Text-based Documents Under Version Control}
-% here.
-%
-% It's a different style than the rest of the paper, and even though
-% we're describing Google docs as 'good enough' it comes across as
-% being 'less than because you're not good enough to use these tools',
-% which is not what we're trying to say.
-%
-% GVW: I think this is likely to trigger a lot of disagreement, so
-% it's worth spelling out our reasoning for this particular pair of
-% recommendations.  In particular, I think the two quotes are
-% necessary to show ``true believers'' what the world looks like
-% to rank-and-file researchers.
-
-% \subsection*{Why Two Recommendations for Manuscripts?}
-
-% The first draft of this paper recommended always using plain text in
-% version control to manage manuscripts, but several reviewers pushed
-% back forcefully. For example, Stephen Turner wrote:
-
-% \begin{quote}
-% {\ldots}try to explain the notion of compiling a document to an
-% overworked physician you collaborate with. Oh, but before that, you
-% have to explain the difference between plain text and word
-% processing. And text editors. And markdown/LaTeX compilers. And
-% BiBTeX. And Git. And GitHub. Etc. Meanwhile he/she is getting paged
-% from the OR{\ldots}
-
-% {\ldots}as much as we want to convince ourselves otherwise, when you
-% have to collaborate with those outside the scientific computing
-% bubble, the barrier to collaborating on papers in this framework is
-% simply too high to overcome. Good intentions aside, it always comes
-% down to ``just give me a Word document with tracked changes,'' or
-% similar.
-% \end{quote}
-
-% Similarly, Arjun Raj said in \withurl{a blog
-%   post}{http://rajlaboratory.blogspot.ca/2016/03/from-over-reproducibility-to.html}:
-
-% \begin{quote}
-% Google Docs excels at easy sharing, collaboration, simultaneous
-% editing, commenting and reply-to-commenting. Sure, one can approximate
-% these using text-based systems and version control. The question is
-% why anyone would like to{\ldots}
-
-% The goal of reproducible research is to make sure one
-% can{\dots}reproduce{\ldots}computational analyses. The goal of version
-% control is to track changes to source code. These are fundamentally
-% distinct goals, and while there is some overlap, version control is
-% merely a tool to help achieve that, and comes with so much overhead
-% and baggage that it is often not worth the effort.
-% \end{quote}
-
-Collaborative editing in something like Google Docs is often therefore
-an effective approach. This solution meets the requirements that we've
-outlined above, although you do lose some benefits of text-based
-formats (notably, being able to use non-proprietary tools for editing,
-sharing, and viewing and being amenable to version control). We still
-recommend \emph{against} traditional desktop tools like LibreOffice
-and Microsoft\textregistered Word because they make collaboration more
-difficult than necessary.
+\subsection*{Discussion}
```

Comment:
Calling this subsection 'Discussion' may throw of readers in thinking this is the discussion section of the entire paper. In the first submission, this section was called "Why Two Recommendations for Manuscripts?”. Can we keep that?